testing.py

```diff
@@ -1024,9 +1023,9 @@ def queue_response(self, response_code, media_type="text/html",
                 headers[k.lower()] = v
         self.responses.append((response_code, headers, content))
 
-    def do_get(self, url, headers, **kwargs):
+    def do_get(self, url, *args, **kwargs):
         self.requests.append(url)
-        return self.responses.pop()
+        return self.responses.pop(0)
```

Comment:
Plucking from the front to make loading test response queuing a mite simpler.

Reply:
This is the right thing to do but it's probably going to affect circulation and content as well.